<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_database">
       <attribute name="title">
        <string>Database</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tbl_database">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_navigation">
          <item>
           <widget class="QPushButton" name="btn_editDiscs">
            <property name="text">
             <string>Discs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_editSongs">
            <property name="text">
             <string>Songs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_editBands">
            <property name="text">
             <string>Bands</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_editSingers">
            <property name="text">
             <string>Singers</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_filter">
       <attribute name="title">
        <string>Filter</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="hl_discFilter" stretch="0">
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="1,4">
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="0" column="1">
             <widget class="QComboBox" name="cb_discFilter">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_discFilter">
              <property name="text">
               <string>Disc:</string>
              </property>
              <property name="indent">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="chk_discRequired">
              <property name="text">
               <string>Required</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_songFilter" stretch="1,4">
          <item>
           <widget class="QLabel" name="lbl_songFilter">
            <property name="text">
             <string>Song:</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_songFilter">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_authorFilter" stretch="1,4">
          <item>
           <widget class="QLabel" name="lbl_authorFilter">
            <property name="text">
             <string>Author:</string>
            </property>
            <property name="indent">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_authorFilter">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_applyButton">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_applyFilter">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDiscs"/>
    <addaction name="actionSongs"/>
    <addaction name="actionBands"/>
    <addaction name="actionSingers"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUserManual"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionDiscs">
   <property name="text">
    <string>Discs</string>
   </property>
  </action>
  <action name="actionSongs">
   <property name="text">
    <string>Songs</string>
   </property>
  </action>
  <action name="actionBands">
   <property name="text">
    <string>Bands</string>
   </property>
  </action>
  <action name="actionSingers">
   <property name="text">
    <string>Singers</string>
   </property>
  </action>
  <action name="actionUserManual">
   <property name="text">
    <string>User manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
